src/pages/validator/amplifier/verifier-governance-proposal.mdx

```diff
@@ -0,0 +1,166 @@
+1. [A proposal to create a new contract](/validator/amplifier/verifier-governance-proposal#proposal-to-create-a-new-contract)
+1. [A proposal to instantiate a new contract](/validator/amplifier/verifier-governance-proposal#proposal-to-instantiate-a-new-contract)
+
+## Prerequisites
+
+To create and submit a governance proposal, make sure you have:
+
+- Access to the Amplifier testnet
+- An account with enough funds to create a proposal
+- An updated version of `axelard` with `wasm` enabled. If you don’t have this, run `make WASM=true build`.
+
+## Set up your environment
+
+1. Set the testnet parameters to environment variables:
+
+    ```bash
+    TESTNET=...
+    RPC=http://...
+    GOVERNANCE=axelar10d07y265gmmuvt4z0w9aw880jnsr700j7v9daj
+    ```
+
+1. Run the optimizer to get the contract’s bytecode. This should be done via a [Docker container](https://www.docker.com/) to ensure that the bytecode is built with the proper environment. The following command builds the contract and saves it to the `artifacts` folder:
+
+    ```bash
+    docker run --rm -v "$(pwd)":/code --mount type=volume,source="$(basename "$(pwd)")_cache",target=/code/target --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry cosmwasm/workspace-optimizer-arm64:0.12.13
+    ```
+
+## Proposal to change upload permission to governance address only
+
+1. Create a proposal in the JSON file format. For example:
+
+    ```bash
+    {
+    "title": "set upload access address to axelar10d07y265gmmuvt4z0w9aw880jnsr700j7v9daj",
+    "description": "set wasm code upload access to axelar10d07y265gmmuvt4z0w9aw880jnsr700j7v9daj",
+    "changes": [
+        {
+        "subspace": "wasm",
+        "key": "uploadAccess",
+        "value": {
+            "permission": "AnyOfAddresses",
+            "address": "",
+            "addresses": ["axelar10d07y265gmmuvt4z0w9aw880jnsr700j7v9daj"]
+        }
+        }
+    ],
+    "deposit": "200000000uaxl"
+    }
+    ```
+
+1. Save the JSON proposal to a file, such as `gov-proposal.json`.
+
+1. In your terminal, create a `GOV_PROPOSAL` variable and set its value to the path of your JSON file:
+
+    ```bash
+    GOV_PROPOSAL=/Users/you/Projects/Axelar/gov-proposal.json
+    ```
+
+1. Submit the proposal:
+
+    ```bash
+    axelard tx gov submit-proposal param-change $GOV_PROPOSAL --from test --output json --gas auto --gas-adjustment 2 --gas-prices=0.00005$DENOM --node $RPC --chain-id $TESTNET
+    ```
+
+1. Set the PROPOSAL_ID based on the submission output:
+
+    ```bash
+    PROPOSAL_ID=3
+    ```
+
+1. Ask other verifiers on the [Axelar Discord](https://discord.com/invite/axelar), Telegram groups, and [Forum](https://community.axelar.network/) to vote on your proposal. You can also use other accounts to vote for your own proposal.
+
+## Proposal to create a new contract
```

Comment:
Originally called a "proposal to upload code," but I think that "proposal to create a new contract" sounds clearer.